official/resnet/README.md

```diff
@@ -80,7 +80,22 @@ The appropriate distribution strategy is chosen based on the `--num_gpus` flag.
 By default this flag is one if TensorFlow is compiled with CUDA, and zero
 otherwise.
 
-num_gpus:
+It is also possible to run with multiple workers using the experimental
```

Comment:
Is this sufficient? Or does the user need to specify the distribution_strategy flag too?